lib/matplotlib/backends/backend_gtk3agg.py

```diff
@@ -121,4 +121,13 @@ def new_figure_manager_given_figure(num, figure):
 
 FigureCanvas = FigureCanvasGTK3Agg
 FigureManager = FigureManagerGTK3Agg
+Window = backend_gtk3.WindowGTK3
```

Comment:
If #8773 ("backend class") goes in I guess the following are also candidates for automatic exporting...